packages/amplify-graphql-api-construct-tests/src/commands.ts

```diff
@@ -160,3 +173,60 @@ export const updateCDKAppWithTemplate = (cwd: string, templatePath: string): voi
   copySync(templatePath, binDir, { overwrite: true });
   moveSync(path.join(binDir, 'app.ts'), path.join(binDir, `${path.basename(cwd)}.ts`), { overwrite: true });
 };
+
+/**
+ * Helper function to create a gen 1 project with for migration.
+ * 
+ * @param name project name
+ * @param projRoot project root directory
+ * @param schema schema file to use
+ */
+export const createGen1ProjectForMigration = async (
+  name: string,
+  projRoot: string,
+  schema: string,
+): Promise<{
+  GraphQLAPIEndpointOutput: string;
+  GraphQLAPIKeyOutput: string;
+  DataSourceMappingOutput: string;
+}> => {
+  await initJSProjectWithProfile(projRoot, { name });
+  await addApiWithoutSchema(projRoot, { transformerVersion: 2 });
+  await updateApiSchema(projRoot, name, schema);
+  await amplifyPush(projRoot);
+
+  // TODO: can't use feature flag until released in CLI
+  // addFeatureFlag(projRoot, 'graphqltransformer', 'enableGen2Migration', true);
+  // await amplifyPush(projRoot);
```

Comment:
I assume that adding the feature flag will require a second `amplify push`, since the flow we're testing is migration of an existing Gen1 project. Let's add a comment to make sure we remember that when the time comes to uncomment this block.